apis/src/common/tournament_action.rs

```diff
@@ -0,0 +1,16 @@
+use serde::{Deserialize, Serialize};
+use shared_types::TournamentDetails;
+
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+pub enum TournamentAction {
+    Create(Box<TournamentDetails>),
+    Delete(String),
+    AcceptInvitation(String),
+    DeclineInvitation(String),
+    AddToSeries(String),
+    RemoveFromSeries(String),
+    Join(String),
+    Leave(String),
+    Invite(String),
+    Start(String),
```

Comment:
You are right, just FYI: this is always the same: `nanoid` which is just a `String`, so sure we can newtype this. But I would then go ahead and do that in another PR, because it actually touches a lot of places, not just tournaments, but `games` and `challenges`, too.